src/main/java/uk/gov/hmcts/ccd/domain/casestate/EnablingConditionParser.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.ccd.domain.casestate;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Map;
+
+public interface EnablingConditionParser {
+    Boolean evaluate(String enablingCondition, Map<String, JsonNode> caseEventData);
```

Comment:
shouldn't we pass the whole case data here? In theory an enabling condition can refer to fields on the whole case right?

Reply:
Enabling condition is on the case event data only, not on the whole case data for now...